pkg/image/api/validation/validation.go

```diff
@@ -147,3 +149,30 @@ func ValidateImageStreamMapping(mapping *api.ImageStreamMapping) fielderrors.Val
 	}
 	return result
 }
+
+// ValidateImageStreamTag is essentially a no-op.  We don't allow direct creation of istags
+func ValidateImageStreamTag(ist *api.ImageStreamTag) fielderrors.ValidationErrorList {
+	result := fielderrors.ValidationErrorList{}
+	result = append(result, validation.ValidateObjectMeta(&ist.ObjectMeta, true, oapi.MinimalNameRequirements).Prefix("metadata")...)
+
+	return result
+}
+
+// ValidateImageStreamTagUpdate ensures that only the annotations of the IST have changed
+func ValidateImageStreamTagUpdate(newIST, oldIST *api.ImageStreamTag) fielderrors.ValidationErrorList {
+	result := fielderrors.ValidationErrorList{}
+
+	result = append(result, validation.ValidateObjectMetaUpdate(&newIST.ObjectMeta, &oldIST.ObjectMeta).Prefix("metadata")...)
+
+	// ensure that only annotations have changed
+	newISTCopy := *newIST
+	oldISTCopy := *oldIST
+	newISTCopy.Annotations = nil
+	oldISTCopy.Annotations = nil
+	oldISTCopy.Annotations = nil
+	if !kapi.Semantic.Equalities.DeepEqual(&newISTCopy, &oldISTCopy) {
```

Comment:
@smarterclayton Do we have a clean, reflective implementation that can give me back better field level reporting?  If not, I'd prefer to leave this less than friendly message.  Its correct, just nearly unreadable.